examples/switch/switch-checkbox.html

```diff
@@ -0,0 +1,184 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8" />
+  <title>Switch using input[type=&quot;checkbox&quot;] Example | WAI-ARIA Authoring Practices 1.2</title>
+
+  <!--  Core js and css shared by all examples; do not modify when using this template. -->
+  <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+  <link rel="stylesheet" href="../css/core.css">
+  <script src="../js/examples.js"></script>
+  <script src="../js/highlight.pack.js"></script>
+  <script src="../js/app.js"></script>
+
+  <!--  js and css for this example. -->
+  <link href="css/switch-checkbox.css" rel="stylesheet">
+  <script src="js/switch-checkbox.js" type="text/javascript"></script>
+</head>
+<body>
+  <nav aria-label="Related Links" class="feedback">
+    <ul>
+      <li><a href="../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/projects/2">Related Issues</a></li>
+      <li><a href="../../#switch">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+  <h1>Switch using <code>input[type=&quot;checkbox&quot;]</code> Example</h1>
+  <p>
+    This example implements the <a href="../../#switch">Switch design pattern</a> using <code>input[type=&quot;checkbox&quot;]</code> elements and CSS borders to implement the graphical rendering of the state of the switch.  Switches are similar to checkboxes, but should be rendered by assistive technologies as "on" and "off", instead of "checked" and "unchecked".
+  </p>
+  <p>Similar examples include: </p>
+  <ul>
+    <li><a href="switch.html">Switch example using the <code>div</code> element.</a></li>
+    <li><a href="switch-button.html">Switch example using the <code>button</code> element.</a>.</li>
+  </ul>
+
+  <section>
+    <h2 id="ex_label">Example</h2>
```

Comment:
Missing `<div class="example-header">` etc to enable the CodePen button.
https://github.com/w3c/aria-practices/wiki/How-to-add-the-Open-In-Codepen-button-to-an-example-page#steps-to-add-the-button-to-existing-examples

Reply:
@zcorpan 
The code pen button is working now in the preview view